orion-server/src/main/java/com/pinterest/orion/server/api/AMI.java

```diff
@@ -0,0 +1,25 @@
+package com.pinterest.orion.server.api;
+
+public class AMI {
```

Comment:
nit: I'd prefer `Ami` to remain consistent with CamelCase naming.

Reply:
Agree. Done.